packages/story-editor/src/components/library/panes/text/fontPreview.js

```diff
@@ -111,6 +117,15 @@ function FontPreview({ title, element, insertPreset, getPosition, index }) {
 
   const { calculateAccessibleTextColors } = usePageAsCanvas();
 
+  const { isText, updateSelectedElements } = useStory(({ state, actions }) => {
+    const isText =
+      state.selectedElements && areAllType('text', state.selectedElements);
```

Comment:
Nit: This technically needs a Boolean wrapper to be consistently a bool.

```js
    const isText =
      Boolean(state.selectedElements && areAllType('text', state.selectedElements));
```

Reply:
Maybe we can just use `isText: areAllType('text', state.selectedElements),` because in the worst case `selectedElements = STABLE_ARRAY` and `Boolean(STABLE_ARRAY || any array) === true`.

Reply:
This extra `state.selectedElements &&` is for checking if the `state.selectedElements` is set at all, if it's `null` then `areAllType` would error since it's checking for `selectedElements.length` inside. Thoughts?

Reply:
My point is: is it possible that `selectedElements` can be `null`?
https://github.com/GoogleForCreators/web-stories-wp/blob/main/packages/story-editor/src/app/story/storyProvider.js#L96

Reply:
Perhaps not! For some reason I remember it erroring without checking first but doesn't seem to happen currently. Changing.